pages/item-new/_.vue

```diff
@@ -0,0 +1,369 @@
+        agents: null,
+        altTitle: null,
+        cardGridClass: null,
+        concepts: null,
+        coreFields: null,
+        description: null,
+        error: null,
+        fields: null,
+        identifier: null,
+        isShownAt: null,
+        media: [],
+        relatedEntities: [],
+        selectedMediaItem: null,
+        similarItems: [],
+        annotations: [],
+        taggingAnnotations: [],
+        transcribingAnnotations: [],
+        title: null,
+        type: null,
+        useProxy: true
+      };
+    },
+
+    computed: {
+      ...mapGetters({
+        apiConfig: 'apis/config'
+      }),
+      keywords() {
+        // Convert collection of annotations' prefLabels into a single langMap
+        return this.taggingAnnotations.reduce((memo, annotation) => {
+          for (const lang in annotation.body.prefLabel) {
+            if (!memo[lang]) memo[lang] = [];
+            memo[lang] = memo[lang].concat(annotation.body.prefLabel[lang]);
+          }
+          return memo;
+        }, {});
+      },
+      fieldsAndKeywords() {
+        return { ...this.fields, ...{ keywords: this.keywords } };
+      },
+      allMetaData() {
+        return { ...this.coreFields, ...this.fieldsAndKeywords };
+      },
+      europeanaAgents() {
+        return (this.agents || []).filter((agent) => agent.about.startsWith(`${this.apiConfig.data.origin}/agent/`));
+      },
+      europeanaConcepts() {
+        return (this.concepts || []).filter((concept) => concept.about.startsWith(`${this.apiConfig.data.origin}/concept/`));
+      },
+      europeanaEntityUris() {
+        const entities = this.europeanaConcepts.concat(this.europeanaAgents);
+        return entities.map((entity) => entity.about).slice(0, 5);
+      },
+      titlesInCurrentLanguage() {
+        let titles = [];
+
+        const mainTitle = this.title ? langMapValueForLocale(this.title, this.$i18n.locale) : '';
+        const alternativeTitle = this.altTitle ? langMapValueForLocale(this.altTitle, this.$i18n.locale) : '';
+
+        const allTitles = [].concat(mainTitle, alternativeTitle).filter(Boolean);
+        for (let title of allTitles) {
+          for (let value of title.values) {
+            titles.push({ 'code': title.code, value });
+          }
+        }
+
+        return titles;
+      },
+      descriptionInCurrentLanguage() {
+        if (!this.description) {
+          return false;
+        }
+        return langMapValueForLocale(this.description, this.$i18n.locale);
+      },
+      metaTitle() {
+        return this.titlesInCurrentLanguage[0] ? this.titlesInCurrentLanguage[0].value : this.$t('record.record');
+      },
+      metaDescription() {
+        if (!this.descriptionInCurrentLanguage) return '';
+        return this.descriptionInCurrentLanguage.values[0] ? this.descriptionInCurrentLanguage.values[0] : '';
+      },
+      isRichMedia() {
+        return isRichMedia(this.selectedMedia);
+      },
+      selectedMedia: {
```

Comment:
All this logic for selectedMedia has been moved into the ItemHero component too. It's still needed for some of the metadata here, but I don't believe this to actually be valuable. There's no point in changing the og metadata for the image when a user changes the image as when shared whatever service the item is shared to will simply retrieve the page again, thereby selecting the default media for the og metadata.

We will need to have the page be aware of when download has been disabled for the current media, but that hasn't been implemented on the ItemHero yet either.